scenarios/openshift/pod_network_shaping.yml

```diff
@@ -0,0 +1,14 @@
+# yaml-language-server: $schema=../plugin.schema.json
+- id: pod_network_shaping
```

Comment:
s/pod_network_shaping/pod_egress_shaping

Reply:
Thanks for the review, fixed in the latest commit.